js/apps/admin-ui/src/sessions/SessionsTable.tsx

```diff
@@ -50,9 +52,20 @@ export type SessionsTableProps = {
 
 const UsernameCell = (row: UserSessionRepresentation) => {
   const { realm } = useRealm();
+  const { t } = useTranslation();
   return (
     <Link to={toUser({ realm, id: row.userId!, tab: "sessions" })}>
       {row.username}
+      {row.transientUser && (
+        <Tooltip content={t("transientUserTooltip")}>
+          <Label
+            data-testid="user-details-label-transient-user"
+            icon={<InfoCircleIcon />}
+          >
+            {t("transientUser")}
+          </Label>
+        </Tooltip>
```

Comment:
A compact label would look better (a space added for visual separation as well)
```suggestion
        <>
          {' '}
          <Tooltip content={t("transientUserTooltip")}>
            <Label
              data-testid="user-details-label-transient-user"
              icon={<InfoCircleIcon />}
              isCompact
            >
              {t("transientUser")}
            </Label>
          </Tooltip>
        </>
```